package.json

```diff
@@ -58,7 +58,7 @@
     "@types/long": "^4.0.1",
     "@types/memory-fs": "^0.3.2",
     "@types/xmldoc": "^1.1.5",
-    "brs": "^0.43.0",
+    "brs": "^0.45.0",
```

Comment:
Just to double check, since you mention it in the title of the pull request: do you want to use the `RokuCommunity` package of `brs` or the original (Sean B's) package of `brs`?

If the former, we should adjust the package name to: [`@rokucommunity/brs`](https://www.npmjs.com/package/@rokucommunity/brs) versus [`brs`](https://www.npmjs.com/package/brs).